changelog.md

```diff
@@ -2,11 +2,13 @@
 
 //  Add your own contribution below
 
+* Add support for failing the build without outputting any text - mxstbr
```

Comment:
This should probably be something more like:

> Builds which only use `markdown` now only show the markdown, and no violations table is shown - mxstbr

Which makes a bit more sense without the context of this PR 👍